### PR TITLE
Standardize readmes for all receivers

### PR DESCRIPTION
**Pull request by djaglowski**

### CODE REVIEW

File: receiver/httpdreceiver/README.md [hunk 1]
```diff
@@ -1,42 +1,40 @@
...
   httpd:
     endpoint: "http://localhost:8080/server-status?auto"
     collection_interval: 10s
 ```
+
+The full list of settings exposed for this receiver are documented [here](./config.go) with detailed sample configurations [here](./testdata/config.yaml).
+
+## Metrics
+
+Details about the metrics produce by this receiver can be found in [metadata.yaml](./metadata.yaml)
```
Comment (dehaansa) [inline receiver/httpdreceiver/README.md:40]: produce*d*

**Review by dehaansa — APPROVED** [review]

File: receiver/postgresqlreceiver/testdata/config.yaml [hunk 2]
```diff
@@ -1,20 +1,20 @@
 receivers:
   postgresql:
-    collection_interval: 10s
+    endpoint: 127.0.0.1:5432
     username: otel
-    password: otel
+    password: $POSTGRESQL_PASSWORD
     database: otel
-    endpoint: 127.0.0.1:5432
```
Comment (JonathanWamsley) [inline receiver/postgresqlreceiver/testdata/config.yaml:7]: Can replace endpoint 127.0.0.1 with local host

File: receiver/postgresqlreceiver/README.md [hunk 3]
```diff
@@ -1,45 +1,40 @@
...
-`ns`, `us` (or `µs`), `ms`, `s`, `m`, `h`.
-
-Example:
+### Example Configuration
 
 ```yaml
 receivers:
   postgresql:
-    collection_interval: 10s
+    endpoint: 127.0.0.1:5432
```
Comment (JonathanWamsley) [inline receiver/postgresqlreceiver/README.md:29]: Can replace endpoint 127.0.0.1 with local host

Comment (JonathanWamsley) [inline receiver/postgresqlreceiver/README.md:29]: Can replace endpoint 127.0.0.1 with localhost

**Pull request merged**